docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
A batch file? Will this work on linux based systems?

Reply:
Yes it is there in case a Windows user wants to generate the documentation. Linux based systems users would use the Makefile. This structure is generated like that by sphinx.